tests/active-active-rhel7-proxy/versions.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.38"
+      version = "~> 3.72.0"
```

Comment:
I believe we wanted to avoid using the patch version, so maybe we can just do `~> 3.72` here.